modules/combined/test/tests/catch_release/catch_release.i

```diff
@@ -39,20 +40,20 @@
   [../]
 []
 
-[SolidMechanics]
-  [./solid]
-    disp_x = disp_x
-    disp_y = disp_y
-    disp_z = disp_z
+[Kernels]
+  [./TensorMechanics]
+    use_displaced_mesh = true
```

Comment:
I would strongly suggest to set these tests up with the Master action rather than the old deprecated action.